lib/frodo/service.rb

```diff
@@ -70,7 +70,11 @@ def entity_sets
     # @param entity_set_name [to_s] the name of the EntitySet desired
     # @return [Frodo::EntitySet] an Frodo::EntitySet to query
     def [](entity_set_name)
-      entity_container[entity_set_name]
+      if options.key?(:with_metadata) && !options[:with_metadata]
```

Comment:
So if options Has a key and the key is False we will do the new logic.

If options has a key and the key is true we do the old logic.

If the options does not have a key we do the old logic. -> Shouldn't this case be defaulting to the new logic?